test/greet-test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const expect = require('chai').expect;
+const greet = require('../lib/greet.js');
+
+describe('testing greet module', function(){
+  describe('valid inputs', function() {
+    it('should return "Hello gnoevil"', function(){
+      expect(greet('gnoevil')).to.equal('Hello gnoevil');
+    });
+  });
+  describe('invalid inputs', function(){
+    it('should throw a missing name error', function() {
+      expect(greet(13)).to.throw(Error);
```

Comment:
You don't actually throw an error here and your test fails because of this. You set your ``greet`` function to return null if someone passes in an arg that is not a string, therefore you can have an expect statement that says:

```
expect(greet(13)).to.equal(null);
```

And your test will pass.